composer.json

```diff
@@ -13,14 +13,12 @@
     "require" : {
         "php" : ">=5.3",
         "symfony/framework-bundle": ">=2.3,<3.0",
-        "ftrrtf/rollbar-notifier": "~1.0"
+        "ftrrtf/rollbar-notifier": "~1.0",
+        "twig/extensions": "^1.1"
```

Comment:
Twig is not a mandatory library (and should not be so), it generally can not be installed in a project (e.g. some API).